### DISCUSSION TITLE
marcus-json 1.0.1发布

### DISCUSSION THREAD

**Discussion by JsonMs** [JsonMs/marcus-json#1] [Announcements]
## Marcus Json 功能介绍
1. 支持Jackson和fastjson的基础实现
2. 支持类型JSONPath、fastjson、Jackson、json read/set
3. 支持基础的注解指定自定义类型
4. 结构api、处理器、基础底座分层处理
5. 支持springboot启动器自动配置
6. 支持spi扩展方式，文件和注解方式

## 后续功能 ...
- 支持json和xml的转化
- 支持json的mock能力
- 动态切换jar包，支持热部署

**Comment by JsonMs** [comment]
上传中央仓库
Springboot地址
https://search.maven.org/artifact/io.github.jsonms/marcus-json-spring-boot-starter/1.0.1/jar
基础API方式
https://search.maven.org/artifact/io.github.jsonms/marcus-json-api/1.0.1/jar

**Reply by konitzmarcus5-stack** [reply]
Atsch
